### PR TITLE
Bootstrap Chain

### PR DESCRIPTION
**Pull request by nunojusto**
## Problem

Solving issue https://github.com/CityOfZion/neo-local/issues/87

## Solution

A switch in the .env file to choose what chain to bootstrap/use.
Those images must be already built.

## Checklist

- [x] I have ran the tests locally.
- [x] I branched off of the `develop` branch and not `master`.
- [x] I did not change the `VERSION` file.

### CODE REVIEW

File: cli/stack/postgres.go [hunk 1]
```diff
@@ -17,8 +17,13 @@ func NewPostgres() Service {
 			},
 			ExposedPorts: map[nat.Port]struct{}{"5432/tcp": {}},
 		},
+<<<<<<< HEAD
```
Comment (revett) [inline cli/stack/postgres.go]: Please update branch to be clean.

File: docker-compose.yml [hunk 2]
```diff
@@ -13,23 +13,65 @@ services:
     ports:
       - "4002:4002"
 
-  neo-nodes:
-    container_name: neo-nodes
-    image: cityofzion/neo-privatenet:2.7.6
+  neo-privatenet1:
+    container_name: neo-privatenet1
+    image: cityofzion/neo-local-privatenet:$NODE1_NEO_CLI_VERSION$BOOTSTRAP
+    #build:
```
Comment (revett) [inline docker-compose.yml]: Can these comments be removed.

File: docker-compose.yml [hunk 3]
```diff
@@ -13,23 +13,65 @@ services:
     ports:
       - "4002:4002"
 
-  neo-nodes:
-    container_name: neo-nodes
-    image: cityofzion/neo-privatenet:2.7.6
+  neo-privatenet1:
```
Comment (revett) [inline docker-compose.yml]: Can the naming format be:

```
neo-privatenet-x
```

File: documentation/README.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
+<p align="center">
```
Comment (revett) [inline documentation/README.md]: Please remove this from this PR as it is changes that are also in another PR.

**Comment by nunojusto** [comment]
Closing this and doing a clean PR with
https://github.com/CityOfZion/neo-local/pull/115

**Pull request closed without merging**